README.md

```diff
@@ -14,6 +14,16 @@ The NVIDIA MONAI Cloud APIs offer a comprehensive suite of tools designed to enh
    - A robust foundation model, trained on 117 classes encompassing various organs and tumors.
    - The model's training and validation involved CT images from over 4,000 patients, ensuring its high accuracy and reliability for medical image analysis.
 
+4. **VISTA-2D Model**:
+   - VISTA-2D is a state-of-the-art model specifically designed for cell image segmentation.
+   - Built on the flexible Meta SAM architecture and adapted for cell imaging, VISTA-2D outperforms the current leading model, Cellpose 2.0. It can be fine-tuned on customer-specific data to deliver unparalleled accuracy in segmenting cells and employs a more generalized approach that adapts more easily to different cell types.
+
+5. **MAISI Model**
+   - The Synthetic Data Generation Model is a state-of-the-art generative AI model that creates high-quality 3D CT images with or without anatomical annotations.
```

Comment:
We say many "state-of-the-arts", sounds like too much...

Reply:
Thanks @tangy5 . That's a good suggestion! I am changing them to `advanced` and `cutting-edge` now.